src/calendar/header/index.tsx

```diff
@@ -30,12 +30,17 @@ import styleConstructor from './style';
 import {Theme, Direction} from '../../types';
 
 export interface CalendarHeaderProps {
+  month?: XDate;
```

Comment:
Should be replaced by string as well?

Reply:
It should be a private prop... We'll need to sort all these props issues in V2, it's currently a big mess